pkg/services/govmomi/service_test.go

```diff
@@ -140,6 +144,11 @@ func Test_reconcileIPAddressClaims_ShouldGenerateIPAddressClaims(t *testing.T) {
 		claimedCondition := conditions.Get(ctx.VSphereVM, infrav1.IPAddressClaimedCondition)
 		g.Expect(claimedCondition).NotTo(BeNil())
 		g.Expect(claimedCondition.Reason).To(Equal(infrav1.WaitingForIPAddressReason))
+
+		// Ensure that the claims have a reference to the cluster to support pauses
+		for _, claim := range ipAddrClaims.Items {
+			g.Expect(claim.ObjectMeta.Annotations).To(HaveKeyWithValue(clusterv1.ClusterLabelName, "my-cluster"))
+		}
```

Comment:
This can be moved under https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/6c50469d4c890ad32e39bdd81c3c0aa932e16f97/controllers/vspherevm_ipaddress_reconciler_test.go#L90-L91